src/pages/api/user.js

```diff
@@ -49,19 +49,21 @@ function createUser(req, res) {
 }
 
 function getUser(req, res) {
-  console.log("QUERY", req.query);
-  // const reqUser = req.user
-  // console.log(reqUser)
-  User.find({ username: req.query.email })
-    .then((user) => {
-      console.log("TEST", req.query.email);
-      console.log("USER", user);
-      res.send(user);
-    })
-    .catch((err) => console.log(err));
+  const { email } = req.query;
```

Comment:
I noticed that each of our endpoints for the User actions are pulling parameters out of different sections in the request. We may want to consider standardizing our request format to make these consistent.